lib/resources/r4/condition.yaml

```diff
@@ -0,0 +1,119 @@
+  binding:
+    description: The verification status to support or decline the clinical status of the condition or diagnosis.
+    terminology:
+      - display: Condition Verification Status Codes
+        system: https://terminology.hl7.org/CodeSystem/condition-ver-status
+        info_link: https://hl7.org/fhir/r4/valueset-condition-ver-status.html
+
+- name: category
+  required: 'No'
+  cardinality: 0..*
+  type: CodeableConcept
+  description: problem-list-item | encounter-diagnosis
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.category
+  example: |
+    {
+      "category": {
+        "coding": [
+          {
+            "system": "https://terminology.hl7.org/CodeSystem/condition-category",
+            "code": "problem-list-item",
+            "display": "Problem List Item"
+          }
+        ],
+        "text": "Diagnosis"
+      }
+    }
+  binding:
+    description: A category assigned to the condition.
+    note: Category code `problem-list-item`is supported for search functionality.
+    terminology:
+      - display: Condition Category Codes
+        system: https://terminology.hl7.org/CodeSystem/condition-category
+        info_link: https://hl7.org/fhir/r4/valueset-condition-category.html
+
+- name: severity
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Severe | Moderate | Mild
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.severity
+  example: |
+    {
+      "severity": [
+        {
+          "coding": [
+            {
+              "system": ""https://snomed.info/sct",
+              "code": "255604002",
+              "display": "Mild"
+            }
+          ]
+        }
+      ]
+    }
+  example2: |
+    {
+      "type": [
+        {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/12022",
+              "code": "3293",
+              "display": "Mild"
+            }
+          ]
+        }
+      ]
+    }
+  binding:
+    description: A subjective assessment of the severity of the condition as evaluated by the clinician.
+    terminology:
+      - display: SNOMED CT
+        system: https://snomed.info/sct
+        info_link: https://hl7.org/fhir/r4/snomedct.html
+      - display: Millennium Condition Severity
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/12022
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes/#code-set-12022-severity
+
+- name: code
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Identification of the condition, problem or diagnosis.
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.code
+  binding:
+    description: Identification of the condition, problem or diagnosis.
+    terminology:
+      - display: SNOMED CT
+        system: https://snomed.info/sct
```

Comment:
snomed, icd 9 and icd 10 should have http:// instead of https://

https://hl7.org/fhir/r4/icd.html

Reply:
Fixed. throughout. [diff](https://github.com/cerner/fhir.cerner.com/pull/273/commits/87ed58e3a827f1d8f8be7ceace048f199170fc55#diff-abb99d2139d79f8a2a8f1d214f5f6882R112)